README.md

```diff
@@ -9,6 +9,20 @@
 The example plugin prints "example" on every query that go handled by the server. It serves as
 documentation for writing CoreDNS plugins.
 
+## Compilation
+
+This package will always be compiled as part of CoreDNS and not in a standalone way. It will require you to use ''go get'' or as a dependency on [plugin.cfg](https://github.com/coredns/coredns/blob/master/plugin.cfg).
+
+The [manual](https://coredns.io/manual/toc/#what-is-coredns) will have more information about how to configure and extend the server with external plugins.
+
+A simple way to consume this plugin, is by adding the following on [plugin.cfg](https://github.com/coredns/coredns/blob/master/plugin.cfg), and recompile it as defined on the [manual](https://coredns.io/manual/toc/#writing-plugins):
+
+~~~
+example:github.com/coredns/example
+~~~
+
+This is also well explained on this [blog post](https://coredns.io/2017/03/01/how-to-add-plugins-to-coredns).
```

Comment:
Remove this and add the steps here (again) just so you don't need to go to another website.

Reply:
I'm replacing that line by this.

I will amend my original commit and push -f:

After this you can compile coredns by:

```shell script
go generate
go build
```

Or you can instead use make:

```shell script
make
```

Reply:
I pushed -f... take a look if that's a good style or if you also want an amend here.


Thanks again.